test/test_utils.erl

```diff
@@ -75,6 +77,18 @@ uplink_packet_up(Opts0) ->
     SigFun = maps:get(sig_fun, Opts0, fun(_) -> <<"signature">> end),
     hpr_packet_up:sign(PacketUp, SigFun).
 
+-spec packet_route(Opts :: map()) -> hpr_packet_route:route().
+packet_route(Opts0) ->
+    NetID = maps:get(net_id, Opts0, 0),
+    DevAddrRanges = maps:get(devaddr_ranges, Opts0, [
+        #packet_router_route_devaddr_range_v1_pb{start = 1, 'end' = 10}
+    ]),
+    EUIs = maps:get(euis, Opts0, [#packet_router_route_eui_v1_pb{app_eui = 1, dev_eui = 1}]),
```

Comment:
```suggestion
    DevAddrRanges = maps:get(devaddr_ranges, Opts0, [{1, 10}]),
    EUIs = maps:get(euis, Opts0, [{1, 1}]),
```
It wasn't breaking because the list comprehension was discarding in `hpr_route:new/6` was bypassing the values that didn't match. Defaults here would always result in empty EUI and DevAddr lists.